soap/soap.php

```diff
@@ -261,7 +262,7 @@ public function SoapClient ($wsdl, array $options = null) {}
      * @throws SoapFault A SoapFault exception will be thrown if the wsdl URI cannot be loaded.
      * @since 5.0.1
      */
-    public function __construct ($wsdl, array $options = null) {}
+	public function __construct (#[LanguageLevelTypeAware(["7.0" => "string"], ["7.1" => "?string"], default: "")] $wsdl, array $options = null) {}
```

Comment:
Please note that not a set of arrays but array of key => value should be used instead. `#[LanguageLevelTypeAware(["7.0" => "string", "7.1" => "?string"], default: "")`.